### DISCUSSION TITLE
Exculed certain rss feeds from 'All' section

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [spacecowboy/Feeder#202] [Ideas]
I have certain feeds which creates lots of noise ( arch packages ), i would still like to view them but other time at certain pace. 

Do you think it would be good addtion to app where we can exclude rss feeds from the all section. 

it would be opposite of the meaning 'all' , maybe instead we can create custom  sections. That would be good too. But then again we could already do this using 'tags'.

i am little bit confuse. What are your thoughts?

**Comment by spacecowboy** [comment]
See discussion at https://github.com/spacecowboy/Feeder/pull/185

**Reply by KAGEYAM4** [reply]
Do we continue the discussion in the #185 ? I will close this issue if it's yes.

**Reply by spacecowboy** [reply]
no need to close this but please move any discussion there so it's all collected
